src/main/scala/kinesis/mock/Decoder.scala

```diff
@@ -0,0 +1,25 @@
+package kinesis.mock
+
+import io.bullet.borer
+import io.circe
+
+trait Decoder[A] {
```

Comment:
Since AWS seems to use different date-time formats for CBOR vs JSON, I introduced a new Decoder/Encoder trait, which implement two `Circe` instances and derives the `Borer` instances from the compliant circe instance.